src/Uno.UI/UI/Xaml/Controls/Popup/PopupPanel.cs

```diff
@@ -238,7 +238,7 @@ private protected override void OnUnloaded()
 
 	// TODO: pointer handling should really go on PopupRoot. For now it's easier to put here because PopupRoot doesn't track open popups, and also we
 	// need to support native popups on Android that don't use PopupRoot.
-	private void OnPointerPressed(object sender, PointerRoutedEventArgs args)
+	protected virtual void OnPointerPressed(object sender, PointerRoutedEventArgs args)
```

Comment:
`protected` make it available to the public world. If we want to be able to override it internally, then it should be `private protected`